src/components/formFields/input.tsx

```diff
@@ -19,21 +21,25 @@ export const InputPassword: React.FC<IInputProps & IReactHookFormProps> = ({
   register,
   placeholder,
   errors,
+  hideErrorMessage,
 }) => {
   const [showPassword, setShowPassword] = React.useState<boolean>(false);
 
   return (
-    <TextField
-      type={showPassword ? "text" : "password"}
-      {...{ disabled, placeholder }}
-      {...register(name, { ...validation })}
-      invalid={errors[name]}
-      icon={
-        <span onClick={() => setShowPassword(!showPassword)}>
-          {showPassword ? <FontAwesomeIcon icon={faEye} /> : <FontAwesomeIcon icon={faEyeSlash} />}
-        </span>
-      }
-    />
+    <>
+      <TextField
+        type={showPassword ? "text" : "password"}
+        {...{ disabled, placeholder }}
+        {...register(name, { ...validation })}
+        invalid={errors[name]}
+        icon={
+          <span onClick={() => setShowPassword(!showPassword)}>
+            {showPassword ? <FontAwesomeIcon icon={faEye} /> : <FontAwesomeIcon icon={faEyeSlash} />}
+          </span>
+        }
+      />
+      {!hideErrorMessage && <ErrorMessage message={errors[name]?.message} />}
```

Comment:
I think, using this syntax, when the `errors[name]` does not exist (effectively, there's no error in the current field), we're still going to push a `<span>` into the DOM.

The `ErrorMessage` itself should be added based on `errors[name]`.

(Right now, the user wouldn't really ever know that the `span` was added, but if someday this span gets some padding or margin it would break the layout)

A better implementation (I think) would be:

```
{ errors[name] && !hideErrorMessage && <ErrorMessage message={errors[name].message} /> }
```

1. We're checking if `errors[name]` actually exists, if it doesn't we should never be rendering the `ErrorMessage`;
2. We're checking if `!hideErrorMessage`; because if this was `true` we should not be rendering the `ErrorMessage` either (even if the message _does_ exist);
3. If there is an error message and we shouldn't hide it, then we can just show the error message _without_ an optional `?`.

(Step 1. and 2. can be interchanged, doesn't matter all that much I think).